tests/client.test.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import ReviewsList from '../client/src/Components/ReviewsList';
+import sampleData from './sampleData';
+
+configure({ adapter: new Adapter() });
+
+describe('test', () => {
+  let emptyObj;
+  let wrapper;
+
+  beforeEach(() => {
+    emptyObj = {};
+
+    let mostRecentReview = {
+      id: 281,
+      user_name: 'Carmen_Monahan',
+      restaurant_name: 'Schmitt and Sons',
+      rating: 2,
+      delivery_rating: 2,
+      order_rating: 5,
+      update_date: '2018-12-17T08:00:00.000Z',
+    };
+
+    wrapper = shallow(<ReviewsList reviews_number={sampleData.length} reviews={sampleData} num_of_reviews={emptyObj} />);
+  });
+
+  it('renders without crashing', () => {
```

Comment:
I am wondering if this is redundant? The component has to render without crashing in order to pass the tests below. Everything else looks good to me.

Reply:
i guess you are right i will remove that one